docs/configure_rum_android_sdk.md

```diff
@@ -22,6 +22,18 @@ In addition to [tracking views automatically][4], you can also track specific di
       }
    ```
 
+### Add your own performance timing
+
+On top of RUM’s default attributes, you may measure where your application is spending its time with greater flexibility. The `addTiming` API provides you with a simple way to add extra performance timing. The timing measure will be relative to the start of the current RUM view. For example, you can add a timing when your hero image has appeared:
+
+   ```kotlin
+       fun onHeroImageLoaded() {
+           GlobalRum.get().addTiming("hero_image")
+       } 
+   ```
+
+Once the timing is sent, the timing will be accessible as `@view.custom_timings.<timing_name>` (For example, `@view.custom_timings.hero_image`). You must create a measure before graphing it in RUM analytics or in dashboards. 
```

Comment:
should we add a link to the page about custom measure creation as it is done in browser sdk doc?